Sources/PackageDescription/Target.swift

```diff
@@ -14,10 +14,12 @@ import Foundation
 
 /// A target, the basic building block of a Swift package.
 ///
-/// Each target contains a set of source files that are compiled into a module or test suite.
-/// You can vend targets to other packages by defining products that include the targets.
+/// Each target contains a set of source files that Swift Package Manager compiles into a module
```

Comment:
As far as I understand it, SwiftPM does not compile any code. Instead, it tells the driver and compiler what to compile and how to compile them.

Reply:
My view is that that distinction is probably unnecessary to make for users of SwiftPM.

Reply:
That's fair, and perhaps SwiftPM can be considered as the compiler frontend's frontend.

Anyway, I think the current wording is better than the proposed change, since the passive voice hides the distinction, so it's both correct and friendly to the user.

Reply:
Yes. This documentation is aimed at developers who use SwiftPM, and not at developers working on SwiftPM.